src/Controller/ForgotMyPasswordController.php

```diff
@@ -90,17 +91,18 @@ public function reset(Request $request): Response
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            if (\date_diff(new \DateTime('now'), $user->getForgotPasswordTokenTime())->i > 60) {
+            if (date_diff(new \DateTime('now'), $user->getForgotPasswordTokenTime())->i > 60) {
```

Comment:
there are methods on the `\DateTime` class to perform diffs.

don't ever use `\date_*` functions, but instead use their OOP alternatives implemented on the `\DateTime` class